src/controller/python/test/test_scripts/commissioning_test.py

```diff
@@ -39,6 +39,7 @@
 # Network id, for the thread network, current a const value, will be changed to XPANID of the thread network.
 TEST_THREAD_NETWORK_ID = "fedcba9876543210"
 TEST_DISCRIMINATOR = 3840
+TEST_DISCOVERY_TYPE = 2
```

Comment:
is that possible to add another two test with TEST_DISCOVERY_TYPE = 0 and 1

Reply:
you can call `python3 commissioning_test.py --discovery-type 0` or `python3 commissioning_test.py --discovery-type 1`

Reply:
is that possible to automatically run them in CI?

Reply:
> is that possible to automatically run them in CI?

It's OK now

Reply:
thank you! Just approve it.